packages/expo-router/src/global-state/routing.ts

```diff
@@ -257,35 +257,14 @@ function getNavigateAction(
     actionStateRoute = actionStateRoute.state?.routes[actionStateRoute.state?.routes.length - 1];
   }
```

Comment:
We no longer need `__EXPO_ROUTER_key` as this logic is handled inside the router override.